### PR TITLE
Test models

### PR DESCRIPTION
**Pull request by fkurusu**
Just added test cases for video and category models.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -24,6 +24,7 @@ end
 
 group :test do
   gem 'database_cleaner', '1.2.0'
+  gem 'shoulda'
```
Comment (brandoncc) [inline Gemfile]: This gem is for Test::Unit/Minitest. The correct gem for Rspec is `should-matchers`, which you can see in the docs [here](https://github.com/thoughtbot/shoulda-matchers#rspec).

File: spec/models/category_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  it 'validates has_many videos' do
+    have_many(:videos)
+  end
+
+  it "validates prsence of name" do
+    validate_presence_of(:name)
+  end
+
+end
+
```
Comment (brandoncc) [inline spec/models/category_spec.rb]: No need for these extra lines.

File: spec/models/category_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe Category do
+  it 'saves itself' do
```
Comment (brandoncc) [inline spec/models/category_spec.rb]: Now that you are using shoulda matchers, you don't need these manual specs anymore.

File: spec/models/video_spec.rb [hunk 4]
```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+
+describe Video do
+  it "saves itself" do
```
Comment (brandoncc) [inline spec/models/video_spec.rb]: Or these

File: spec/models/category_spec.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+
+describe Category do
+  it { have_many(:videos) }
```
Comment (brandoncc) [inline spec/models/category_spec.rb]: Do these pass? The general syntax is:

```ruby
it { should have_many(:videos) }
```

The `expect` syntax is preferred over `should` syntax now though:

```ruby
it { is_expected.to have_many(:videos) }
```

Comment (fkurusu) [inline spec/models/category_spec.rb]: Yes, they pass without 'should'. So, should I put 'should' or is_expect?

**Pull request merged**